dwave/plugins/sklearn/transformers.py

```diff
@@ -0,0 +1,334 @@
+            SelectFromQuadraticModel: This instance of `SelectFromQuadraticModel`
+        """
+        if number_of_features is None:
+            number_of_features = self.n_default_feature
+
+        if hasattr(X, "toarray") and (not isinstance(X, pd.DataFrame)):
+            X = X.toarray()
+
+        if self.method == "correlation":
+            model_matrix, f_model = self.calculate_correlation_matrix(X, y)
+        elif self.method == "mutual information":
+            if y is None:
+                raise ValueError("mutual infromation requires outcome")
+            model_matrix, f_model = self.calculate_mutual_information(X, y)
+        else:
+            raise ValueError(f"Only methods {self.acceptable_methods} are implimented")
+
+        # feature selection formulation (same as example)
+
+        logging.info(f"constructing feature selection model")
+
+        if self.chunksize is not None:
+            chunksize = self.chunksize
+        else: 
+            chunksize = 100 
+
+        if X.shape[1] < chunksize:
+            logging.info("constructing BQM using dimod constructor from matrix")
+
+            feature_selection_bqm = dimod.BQM(model_matrix, "BINARY")
+        else:
+            logging.info("constructing bqm using chunked iteration")
+
+            feature_selection_bqm = dimod.BQM(vartype="BINARY")
+            feature_selection_bqm.add_variables_from(
+                {i: model_matrix[i, i] for i in range(model_matrix.shape[0])}
+            )
+
+            logging.info("variables and linear biases added")
+
+            for row_index in range(0, model_matrix.shape[0], chunksize):
+                for col_index in range(0, model_matrix.shape[1], chunksize):
+                    row_index_max = min(row_index + chunksize, model_matrix.shape[0])
+                    col_index_max = min(col_index + chunksize, model_matrix.shape[0])
+                    chunk = model_matrix[
+                        row_index:row_index_max, col_index:col_index_max
+                    ]
+                    quad_biases = [
+                        (i[0], i[1], j) for i, j in np.ndenumerate(chunk) if i[0] > i[1]
+                    ]
+                    feature_selection_bqm.add_interactions_from(quad_biases)
+
+        f_model.close()
+
+        logging.info("BQM created")
+
+        feature_selection_cqm = dimod.CQM()
+        feature_selection_cqm.set_objective(feature_selection_bqm)
+        
+        logging.info("CQM created (objective only)")
+
+        sense = "==" if strict else "<="
+
+        feature_selection_cqm.add_constraint_from_iterable(
+            [(var, 1) for var in feature_selection_cqm.variables],
+            sense,
+            number_of_features,
+        )
+
+        try:
+            cqm_solver = LeapHybridCQMSampler()
+        except (ConfigFileError, SolverAuthenticationError) as e:
+            raise RuntimeError(
+                f"""The Leap hybrid solver raised the following error: {e}. 
```

Comment:
@JoelPasvolsky , opinion on what text should be here?

Reply:
Perhaps something like, f"""Instantiation of a Leap hybrid solver failed with an {e} error.
                                             See https://docs.ocean.dwavesys.com/en/stable/overview/sapi.html
                                             for configuring access to Leap’s solvers."""

I wouldn't try diagnosing installation of dwave.system, tokens, etc because (1) they might not have installed locally, they might have pip installed this package into an IDE workspace (2) in that case https://docs.dwavesys.com/docs/latest/doc_ide_user.html might be helpful, but even giving the more-likely link above is a maintenance risk, docs links do change over time, so I hesitate giving even the above versus a generic "See how to configure access to Leap solvers in the Ocean documentation at https://docs.ocean.dwavesys.com/en/stable/index.html "
Where we give such links we try to have CI run linkcheck